### ISSUE TITLE
Improve documentation of orjson

### ISSUE THREAD

**Issue by mjga-trifork** [nazrulworld/fhir.resources#142]
### Description

The current documentation on orjson says:

> What you need to do, just make [orjson](https://pypi.org/project/orjson/) as your project dependency!

Honestly. I don't quite understand, what that means and what I'm supposed to do.

My guess is, that a few lines on what to do and why it works would be plenty.

**Comment by vadi2** [comment]
Agreed - is installing it enough, or should it be `import`ed into a file?
